inductiva/fluids/post_processing/splishsplash.py

```diff
@@ -40,7 +47,7 @@ def convert_vtk_data_dir_to_netcdf(
         xr_dataset.to_netcdf(os.path.join(netcdf_data_dir, f"{file_stem}.nc"))
 
 
-def read_vtk_file_to_xr_dataset(file_path: str, time: float) -> xr.Dataset:
+def read_vtk_file_to_xr_dataset(file_path: str, time: float):
```

Comment:
I see that this required to remove some type annotations in a couple of places.... but it might be fair enough for now....

Reply:
I thought it was fair for now, since we still need to uniformize the outputs of SPH simulations (either xarray or pyvista).

Reply:
you can use forward references for those type annotations, *i.e.*, specify the type annotation as a string literal (`'xr.Dataset'`). More info [here](https://peps.python.org/pep-0484/#forward-references)